voice/methods/conferences/postConferencesConferenceId.md

```diff
@@ -18,6 +18,10 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
 | Parameter       | Description                                                                     | Mandatory |
 |:----------------|:--------------------------------------------------------------------------------|:----------|
 | status          | (optional) Setting the conference status to `completed` ends the conference.    | No        |
+| redirectUrl     | (optional) The URL to send the [conferenceRedirect](../../bxml/callbacks/conferenceRedirect.md)event which will provide new BXML. Not allowed if `state` is `completed`, but required if `state` is `active`    | No        |
+| redirectMethod  | (optional) The HTTP method to use for the request to `redirectUrl` Not allowed if `state` is `completed`    | No        |
+| username        | (optional) The username to send in the HTTP request to `answerUrl` and `diconnectUrl`    | No        |
```

Comment:
Wrong URL names

Reply:
Yikes, copy and paste is too powerful for me